presto-benchmark-driver/src/main/java/io/prestosql/benchmark/driver/BenchmarkQueryRunner.java

```diff
@@ -271,9 +272,9 @@ private long getTotalCpuTime()
         return TimeUnit.NANOSECONDS.toNanos(totalCpuTime);
     }
 
-    private List<URI> getAllNodes(URI server)
+    private List<URI> getAllNodes(URI server, String user)
     {
-        Request request = prepareGet().setUri(uriBuilderFrom(server).replacePath("/v1/service/presto").build()).build();
+        Request request = prepareGet().setHeader(PrestoHeaders.PRESTO_USER, user).setUri(uriBuilderFrom(server).replacePath("/v1/service/presto").build()).build();
```

Comment:
can we use dummy user instead of actual user name? I think any username will do in benchmarks

Reply:
@sopel39 Can we ignore the `user` name specified by the option `--user`?

Reply:
@Lewuathe I think in benchto for a while some dummy name was used and benchmarks still work. However, I think we can keep it as is